rosbag2_storage_default_plugins/include/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.hpp

```diff
@@ -45,6 +45,7 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteWrapper
   SqliteWrapper();
   ~SqliteWrapper();
 
+  bool is_field_exist(const std::string & table_name, const std::string & field_name);
```

Comment:
```suggestion
  bool field_exists(const std::string & table_name, const std::string & field_name);
```
I just feel that this reads better, your mileage may vary.

Reply:
Ok I agree. already renamed in https://github.com/ros2/rosbag2/pull/1090/commits/3eaaf4d177d75589c21d40832337c13e9df37a02